packages/docs/core-concepts/plugins.md

```diff
@@ -365,18 +365,23 @@ When [using pinia alongside Nuxt](../ssr/nuxt.md), you will have to create a [Nu
 import { PiniaPluginContext } from 'pinia'
 import { Plugin } from '@nuxt/types'
 
-function MyPiniaPlugin({ store }: PiniaPluginContext) {
+const MyPiniaPlugin = ({ store }: PiniaPluginContext) => {
```

Comment:
Let's keep the previous syntax

Reply:
My bad, I thought v2 was the live branch for the docs, there is a syntax error in the live version (missing arrow in arrow function https://pinia.vuejs.org/core-concepts/plugins.html#nuxt-js)

<img width="288" alt="Screenshot 2022-03-15 at 20 11 30" src="https://user-images.githubusercontent.com/25274586/158453495-9f7a09de-f1d1-45a3-9818-7b71849af69f.png">

Reply:
Is there a technical reason for it ?